From 8ca996cdc5c25d2cedf8692f9150d5d775cf18f1 Mon Sep 17 00:00:00 2001
From: Quanyang Wang <quanyang.wang@windriver.com>
Date: Tue, 10 Jan 2023 10:50:03 +0800
Subject: [PATCH] s32: serdes: hwconfig: change hwconfig for CVC board.

At CVC board, the serdes lanes configuration is as below:

Serdes0_lane0: Pcie
Serdes0_lane1: PFE2_SGMII
Serdes1_lane0: PFE0_SGMII
Serdes1_lane1: PFE1_SGMII

Upstream-Status: Inappropriate [WR Linux specific change]

Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 arch/arm/mach-s32/s32-cc/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-s32/s32-cc/Kconfig b/arch/arm/mach-s32/s32-cc/Kconfig
index 0b2d68f3a0..5401f3d22b 100644
--- a/arch/arm/mach-s32/s32-cc/Kconfig
+++ b/arch/arm/mach-s32/s32-cc/Kconfig
@@ -60,7 +60,7 @@ config S32CC_HWCONFIG
 	depends on PHY_S32CC_SERDES
 	default "serdes0:mode=xpcs0&xpcs1,clock=ext,fmhz=125;xpcs0_0:speed=2G5;xpcs0_1:speed=2G5;serdes1:mode=xpcs0&xpcs1,clock=ext,fmhz=125;xpcs1_0:speed=2G5;xpcs1_1:speed=2G5" if (!PCI_S32CC && NXP_PFENG)
 	default "serdes0:mode=pcie,clock=ext;pcie0:mode=rc;serdes1:mode=pcie,clock=int;pcie1:mode=ep" if (PCI_S32CC && !NXP_PFENG)
-	default "serdes0:mode=pcie,clock=ext;pcie0:mode=rc;serdes1:mode=xpcs0&xpcs1,clock=ext,fmhz=125;xpcs1_0:speed=2G5;xpcs1_1:speed=2G5" if (PCI_S32CC && NXP_PFENG)
+	default "serdes0:mode=pcie&xpcs1,clock=int,fmhz=100;pcie0:mode=rc;xpcs0_1:speed=1G;serdes1:mode=xpcs0&xpcs1,clock=int,fmhz=100;xpcs1_0:speed=1G;xpcs1_1:speed=1G" if (PCIE_S32GEN1 && FSL_PFENG)
 
 	help
 	  The configuration for the SerDes modules, stored in
-- 
2.36.1

